src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -313,14 +317,12 @@ where childNode.Name.Equals(sessionXmlTagName)
                     var callsiteId = string.Empty;
                     if (child.HasAttribute(Configurations.CallSiteID))
                     {
-                        callsiteId = child.GetAttribute(Configurations.CallSiteID);
+                        containsTraceData = true;
+                        return loadedData;
```

Comment:
so now, we just return an empty list no matter what, but we make a note that there was at least 1 node with trace data?

Reply:
I renamed the variable from `containsTraceData` to `containsLegacyTraceData` to make it clearer. This flag is only used to trigger the toast notification when opening an old workspace.